package.json

```diff
@@ -10,7 +10,7 @@
     "pretest": "npm install",
     "test": "mocha tests --recursive",
     "test-watch": "mocha tests --recursive -w",
-    "embed-start": "node client/coral-embed-stream/dev-server.js"
+    "embed-start": "webpack --config ./client/coral-embed-stream/webpack.config.dev.js && ./bin/www"
```

Comment:
This should target the local install of `webpack` in `node_modules` not the global.